buildroot/bin/backup_configs

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-cp src/openLRSng.ino     src/openLRSng.default
+cp openLRSng/openLRSng.ino     openLRSng/openLRSng.default
```

Comment:
Looking at this, would it make sense to create a config file for pathing and stuff, and include it at the top of all of the scriptlets?

Reply:
We could but it's not really to the tipping point where it saves much effort.  How about maybe tackle that on the auto-deploy Travis revamp ? :)

Reply:
Fair enough.

Reply:
why not 🤷‍♂️